WordPress/gutenberg
# Image block: Revise lightbox UI to remove 'behaviors'

## What?
Now that the concept of [behaviors](https://github.com/WordPress/gutenberg/issues/50029) is no longer shipping in the near term, this PR removes it from the `theme.json`, the Global Styles and the Image block attributes.

## Why?
We want to provide a lightbox feature and get feedback on its implementation, but we also want to do so in a way that's minimal and easy to understand. Addresses #53403 

## How?
Removing all the references to `behaviors`. 

_Note regarding backwards compatibility: We've added backwards compatibility [in the following PR](https://github.com/WordPress/gutenberg/pull/54071), which is meant to be merged at the same time as this one so there's no gap in support for users using the old syntax we're removing in this PR._

## Testing Instructions
1. Open a post or page.
2. Insert an Image block.
3. Make sure that no "Behaviors" panel is preset inside of the "Advanced" panel of the Inspector Controls.
4. Save the post/page
5. Go to the frontend
6. Make sure that the page loads correctly and there is no Lightbox present.
